src/main/java/org/wildfly/security/auth/realm/ldap/UserPasswordCredentialLoader.java

```diff
@@ -114,7 +118,7 @@ public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential
         @Override
         public SupportLevel getEvidenceVerifySupport(final Class<? extends Evidence> evidenceType, final String algorithmName) throws RealmUnavailableException {
             // If it is obtainable, it is verifiable.
-            return getCredentialAcquireSupport(PasswordCredential.class, null);
+            return getCredentialAcquireSupport(PasswordCredential.class, algorithmName);
```

Comment:
Maybe as long as this is getting changed, we should make sure that the evidence type makes sense?  I guess the only evidence type that makes sense here is ``PasswordGuessEvidence``, yes?